src/Senders/CurlSender.php

```diff
@@ -54,6 +55,10 @@ public function __construct($opts)
         if (array_key_exists('ca_cert_path', $opts)) {
             $this->caCertPath = $opts['ca_cert_path'];
         }
+        if (array_key_exists('ip_resolve', $opts)
```

Comment:
Can we add testing for this scenario?

Reply:
I added some tests to ensure that the value is set. I usually try to avoid testing private properties since they are generally assumed to be safe to refactor. So, I added a comment to the test in case it fails in the future because of some sort of refactoring.

Reply:
I approved the PR but it looks like tests are failing.